lib/selectivity/rspec.rb

```diff
@@ -44,25 +44,25 @@ def find_selectivity_input(from, options)
         find(:div, "##{label[:for]}", options)
       end
 
-      def multiselect?(input)
+      def _selectivity_multiselect?(input)
         input.first('.selectivity-multiple-input-container').present?
       end
 
-      def select!(input, item)
+      def _selectivity_select!(input, item)
         input.click
 
         within 'div.selectivity-dropdown' do
-          if (find(:xpath, "//div[@class='selectivity-result-item'][contains(text(), '#{item}')]")).visible?
+          if find('div.selectivity-result-item', text: item).visible?
```

Comment:
It does not work for this:

![image](https://cloud.githubusercontent.com/assets/940/7604684/7375c33c-f946-11e4-8db2-e31de66cd8c7.png)

I'm using the Poltergeist/PhantomJS driver. Using it with:

```ruby
selectivity_select 'Mustang', from: '#car_offer_model_slug'
```